documentation/src/main/asciidoc/topics/ref_protostream_types.adoc

```diff
@@ -19,7 +15,31 @@ unboxed equivalents in the case of primitive types:
 * `java.util.Date`
 * `java.time.Instant`
 
-.Reference
+[discrete]
+== Additional type collections
+The ProtoStream library includes an `Adapter` class that you can use to marshall additional types.
```

Comment:
There isn't a single `Adapter` class. Instead there are several adaptor classes available for common Java types. For example the `HashSetAdapter`.